app/controllers/posts_controller.py

```diff
@@ -94,7 +94,7 @@ def loadNewsFeed():
   return recentPosts, array_of_comments, proper_date      
 
 
-def handlePost(page_to_load, request): 
+def handlePost(request):
```

Comment:
Since the `@app.route` that calls this function no longer has a `<page_to_load>`, there isn't any to pass to this function.